ci/packages/suite-web.yml

```diff
@@ -77,7 +77,7 @@ suite-web deploy dev:
     COMPOSE_FILE: ./docker/docker-compose.suite-ci.yml
     # corresponds with the NextJs assetPrefix. Tells Cypress where is the index of application
     CYPRESS_ASSET_PREFIX: /web
-    CYPRESS_baseUrl: ${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}
+    CYPRESS_baseUrl: ${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}/
```

Comment:
doesn't this break anything? I would expect it to break asset prefix. After this change cypress might see something like 'sldev.cz/branch//web'

Reply:
Ok I rather remove it.

Reply:
Done. I have force-pushed directly, hopefully you'll forgive me.